test/e2e/scenario_config-envs_test.go

```diff
@@ -94,21 +95,20 @@ func TestConfigEnvs(t *testing.T) {
 	testEnvName := "TEST_ENV"
 	testEnvValue := "TEST_VALUE"
 
-	knFunc := NewTestShellInteractiveCmdRunner(t)
-	knFunc.TestShell.ShouldDumpOnSuccess = false
-	knFunc.commandSleepInterval = time.Millisecond * 1500
+	knFunc := common.NewTestShellInteractiveCmd(t)
+	knFunc.TestCmd.ShouldDumpOnSuccess = false
+	knFunc.CommandSleepInterval = time.Millisecond * 1500
 
 	// On When...
-	project := FunctionTestProject{}
-	project.Runtime = "go"
-	project.Template = "envs"
-	project.FunctionName = "test-config-envs"
-	project.ProjectPath = filepath.Join(os.TempDir(), project.FunctionName)
-	project.RemoteRepository = "http://github.com/boson-project/test-templates.git"
-	project.Builder = builders.Pack
+	funcName := "test-config-envs"
+	funcPath := filepath.Join(t.TempDir(), funcName)
 
-	Create(t, knFunc.TestShell, project)
-	defer func() { _ = project.RemoveProjectFolder() }()
+	knFunc.TestCmd.Exec("create",
+		"--language", "go",
+		"--template", "envs",
+		"--repository", "http://github.com/boson-project/test-templates.git", // TODO Make on config
```

Comment:
I don't really love that we are still using this boson-project repo, but I'm not sure what the best alternative would be. I don't really want to pollute knative-sandbox with more repos if we can avoid it.

Reply:
yea.. I agree!!  I think I have an idea I can try on a different PR.. like using that custom git server (just as oncluster build tests) and pull from there..